framework/auth/__init__.py

```diff
@@ -108,21 +108,24 @@ def register_unconfirmed(username, password, fullname, campaign=None):
     return user
 
 
-def get_or_create_user(fullname, address, is_spam=False):
-    """Get or create user by email address.
+def get_or_create_user(fullname, address, reset_password=True, is_spam=False):
```

Comment:
add `reset_password`:
1. for new user from institution login, no need to ask user to reset password after creation, thus no verification key
2. for conference `add_poster_by_email`, create user and send user confirmation email to reset password, generate reset password verification key